ocdskingfisherarchive/database_archive.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+
+
+class DataBaseArchive:
+
+    def __init__(self, config):
+        self.config = config
+        self.conn = sqlite3.connect(config.database_archive_filepath)
+
+        c = self.conn.cursor()
+        c.execute("SELECT name FROM sqlite_master WHERE type='table' AND name='collections'")
+        if not c.fetchone():
+            c.execute("CREATE TABLE collections (collection_id INTEGER PRIMARY KEY NOT NULL, state TEXT NOT NULL)")
+            c.close()
+            self.conn.commit()
+        else:
+            c.close()
+
+    def get_state_of_collection_id(self, collection_id):
+        state = 'UNKNOWN'
+        c = self.conn.cursor()
+        c.execute("SELECT state FROM collections WHERE collection_id=?", str(collection_id))
```

Comment:
Use named placeholders in SQL queries: https://ocp-software-handbook.readthedocs.io/en/latest/python/code.html#sql-statements

No need to call `str()` in that case (or maybe even in this case).